### DISCUSSION TITLE
What is the current Git Workflow for the project?

### DISCUSSION THREAD

**Discussion by assemblaj** [ikemen-engine/Ikemen-GO#1676] [Q&A]
I'm assuming that broadly speaking, everyone develops on `develop` on their forks, submits PRs to `develop`  and then those are tested and merged into `master` at a later date? How do `release`s factor in?

**Comment by K4thos** [comment]
Already replied on discord but posting here too: https://github.com/ikemen-engine/Ikemen-GO/blob/develop/CONTRIBUTING.md#branching-strategy
